### DISCUSSION TITLE
Two relays boards and 1 m5stick. How?

### DISCUSSION THREAD

**Discussion by toldenburger** [raomin/ESPAltherma#277] [Q&A]
Got myself an m5stickplus, some dupont cables and two relays for controlling the SG1 and 2 contacts. 
But how do I wire up relay numer two since I used G0 on the m5stick for the first relays board. 
Any suggestions? Or do I need a GPIO extender? 

The readme says: ‘ Of course, you will need to use 2 more relays to open/close SG1 and SG2 contacts of your heat pump.’

My apologies
 if I missed this info somwhere.

**Comment by maromme** [comment]
You can use the pins of the groove connector as well (I think it's in the readme somewhere). There you've got to add. GPIOs. Or look at #273

**Comment by Dennis3321** [comment]
I have got the very same question. You will end up with 3 relay's. A schema / drawing / picture would be nice for such a situation.

Is it also possible to use a; M5STACK 4-Relay Unit? A link / picture on what type of relay to use, would also be nice.

**Reply by Dennis3321** [reply]
I've bought and am using a m5stack 4 relay unit. A bit more pricy than the aliexpress route, but it's working. I need 2 relays for the smart grid and I need 1 relay as thermostaat. Had to modify the code to get it to work. I don't know how to share it here if there is any interest...
